cgi/export_products.pl

```diff
@@ -70,6 +70,10 @@
 	$html .= "<p>" . lang("producers_platform_licence") . "</p>";
 	$html .= "<p>" . lang("export_product_data_photos_please_check") . "</p>";
 
+	# Display button for admins only
+
+	if ($User_id =~ /-off$/) {
```

Comment:
Why not use `isAdmin()`?

Reply:
Good point. I'll make things cleaner, I need to implement https://docs.google.com/document/d/154MN36FACbS3PqSaEcsoerSz0n6jo9RwaD3eLRChNgI/edit first though.

Reply:
Good point, I'll make things cleaner. I need to implement https://github.com/openfoodfacts/openfoodfacts-server/issues/2085 first.